share/man/man3/sysexits.3

```diff
@@ -29,7 +29,7 @@
 .Os
 .Sh NAME
 .Nm sysexits
-.Nd preferable exit codes for programs
+.Nd 4.4BSD exit codes for programs
```

Comment:
Again, apologies for multiple comments....  I'd just delete preferable from the original.

Reply:
In the header, I like that, but the document description feels different because of apropos.

How's "s/preferable/sendmail/"? I would use "sendmail style" but I'm worried about it clogging output of "man -k style" when people are trying to learn about writing.